react-ui/package.json

```diff
@@ -28,5 +28,11 @@
   "devDependencies": {
     "@babel/plugin-proposal-decorators": "^7.12.1",
     "jest-environment-jsdom-sixteen": "^1.0.3"
-  }
+  },
```

Comment:
Hi Kyle,
As mentioned before (this may be a side effect of running the code), but please remove the browserslist to prevent deployment issues.

Reply:
Removed